release-notes/7.0/7.0.0/api-diff/Microsoft.NETCore.App/7.0.0_System.Runtime.InteropServices.JavaScript.md

```diff
@@ -0,0 +1,232 @@
+# System.Runtime.InteropServices.JavaScript
+
+``` diff
++namespace System.Runtime.InteropServices.JavaScript {
++    [SupportedOSPlatformAttribute("browser")]
++    public sealed class JSException : Exception {
++        public JSException(string msg);
++    }
++    [AttributeUsageAttribute(AttributeTargets.Method, Inherited=false, AllowMultiple=false)]
++    [SupportedOSPlatformAttribute("browser")]
++    public sealed class JSExportAttribute : Attribute {
++        public JSExportAttribute();
++    }
++    [CLSCompliantAttribute(false)]
++    [EditorBrowsableAttribute(EditorBrowsableState.Never)]
++    [SupportedOSPlatformAttribute("browser")]
++    public sealed class JSFunctionBinding {
++        public JSFunctionBinding();
```

Comment:
This constructor need to be internal. Otherwise `System.Runtime.InteropServices.JavaScript` LGTM

Reply:
```suggestion
+        internal JSFunctionBinding();
```